src/spec/connect/rest.spec.coffee

```diff
@@ -314,8 +314,5 @@ describe 'Rest', ->
           expect(b.message).toBe 'Something went wrong'
           done()
 
-      it 'should throw if limit param is not 0', ->
```

Comment:
But this test is about when limit is NOT 0, it will throw an error. We have test for 0 here: https://github.com/sphereio/sphere-node-sdk/pull/230/files/bf0304245c6c899b56c499f3cf2fc8a21a6bdbf7#diff-0525bb4cabce0587175816bbccaefd26R317